swim/events.go

```diff
@@ -31,7 +31,7 @@ type EventListener interface {
 	HandleEvent(events.Event)
 }
 
-// A MaxPAdjustedEvent occurs when the disseminator adjusts the max propogation
+// A MaxPAdjustedEvent occurs when the disseminator adjusts the max propagation
 // count for changes
```

Comment:
We should run your regex against the entire code base and be done with it for once and for all